docs/DeveloperGuide.md

```diff
@@ -553,6 +576,34 @@ testers are expected to do more *exploratory* testing.
 4. Other incorrect sortevent commands to try: `sortevent x`, `...` (where x is not a, b, c, or d).<br>
    Expected: Similar to previous.
 
+### Editing an Event
+
+1. Editing an event while all events are being shown
+
+    1. Prerequisites: List all events using the `listevent` command. Multiple events in the list.
+
+    2. Test case: `editevent 1 ev/Lana Del Rey Concert`<br>
+       Expected: First event is edited from the event list, such that it now has the new event name. All person(s) with the specified event in their event tags will have that event edited accordingly as well. Details of the edited event shown in the status message.
+
+    3. Test case: `editevent 0`<br>
+       Expected: No event is edited. Error details shown in the status message.
+
```

Comment:
I think you can add one that edits datetime but then the expected results wont expect the event tag in the persons to change (since it only changes when the name is edited)

Reply:
Would need to update the second part below if you're adding this

Reply:
Thanks for your sugggestions! I've edited everything accordingly. For this point, I don't think I'll be making the change since the event tag in the persons will still change in the backend when the dates are edited (just not on the UI display, since we only display the name). Hope that's ok!